xml/System.Threading.Channels/UnboundedChannelOptions.xml

```diff
@@ -33,7 +33,7 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Threading.Channels.UnboundedChannelOptions" /> abstract class. This class cannot be instantiated.</summary>
```

Comment:
Why the difference between "Initializes a new instance" used here and "Protected constructor" used above... why not just always use the "Initializes a new instance" language?

Reply:
I think the "protect constructor" prefix is unnecesary, we can already see the protected keyword in Docs. I'll remove it.